### ISSUE TITLE
1.2.9 Version - Debbuger error in VSCode

### ISSUE THREAD

**Issue by asierf** [Microsoft/vscode-cordova#297]
1.2.8 Version was working fine in windows 7.

ERROR : 
[cordova-tools] Error processing "launch": 'Debugger.setAsyncCallStackDepth' wasn't found

Console : 
'Debugger.setAsyncCallStackDepth' wasn't foundError processing "launch": 'Debugger.setAsyncCallStackDepth' wasn't found

**Comment by iTOYS** [comment]
@asierf, thanks for reporting. Is it cordova or ionic app? 
Could you providing precise steps to reproduce problem(which cordova version using, which configuration did you run, etc.)?
Also, could you please try creating an empty cordova project and try debugging it?

**Comment by asierf** [comment]
It's an Ionic v1 app running under emulated android sdk version 19

Ionic version 3.6.0
Cordova version 7.0.1

"name": "Run Android on emulator",
"type": "cordova",
"request": "launch",
"platform": "android",
"target": "emulator",
 "port": 9222,
 "sourceMaps": true,
  "cwd": "${workspaceRoot}",
  "ionicLiveReload": false

I have the same issue debbuging an empty cordova project

**Comment by vladimir-kotikov** [comment]
Ok, so this happens due to bug in vscode-chrome-debug-core, where `Debugger.setAsyncCallStackDepth` method is called during `attach` and while it is [marked as optional](https://github.com/Microsoft/vscode-chrome-debug-core/blob/012a56f8637a380c2117b6fecd096e5412925d08/crdp/crdp.d.ts#L1308) its existense is [not being checked](https://github.com/Microsoft/vscode-chrome-debug-core/blob/50b5fffe4ebc3edb7bf87cc483eb6ec758d361d5/src/chrome/chromeDebugAdapter.ts#L342) before calling. The workaround for now is to catch the exception appropriately and silently continue

**Comment by vladimir-kotikov** [comment]
@asierf this has been fixed in #298 and will be available in the next release that likely will be available EOW.

**Comment by iTOYS** [comment]
#https://github.com/Microsoft/vscode-chrome-debug-core/issues/226

**Comment by kirajhpang** [comment]
hi, I having this issue on debug ionic v2 when 
`Run iOS on simulator `
`Run iOS on device`
`Attach to running iOS on device/simulator`

Debug Console output message show as following:
```
Attaching to ios
Configuring debugging proxy
Attaching to app.
DEVICE READY FIRED AFTER 2742 ms
Ionic Native: deviceready event fired after 2732 ms
OPEN database: _ionicstorage
new transaction is waiting for open operation
new transaction is waiting for open operation
OPEN database: _ionicstorage - OK
DB opened: _ionicstorage
'Debugger.setAsyncCallStackDepth' was not foundError processing "launch": 'Debugger.setAsyncCallStackDepth' was not found: [object Object]
```

My environment detail:
    Node       : v6.11.1
    OS         : macOS Sierra
    Xcode      : Xcode 8.3.3 Build version 8E3004b
    ios-deploy : 1.9.1
    ios-sim    : 6.0.0
    npm        : 3.10.10

any issue or solution might relate to this?

**Comment by iTOYS** [comment]
Hi @kirajhpang this has been fixed in #298 
Release coming soon
